frontend/src/store/modules/security.ts

```diff
@@ -8,6 +10,10 @@ export default {
             return state.noSecurity;
         },
     },
-    mutations: {},
+    mutations: {
```

Comment:
Bitte wieder zurückbauen. Wenn es jetzt zu einem Fehler beim Laden des UserInfo kommt, wird automatisch immer der NoSecModus aktiviert und jeder kann die Anwendung verwende. Das ist nicht gut in der Produktion